regulations/views/preamble.py

```diff
@@ -231,6 +232,28 @@ def get(self, request, *args, **kwargs):
                                 context=context)
 
 
+class ChromePreambleSearchView(chrome.ChromeSearchView):
+    template_name = 'regulations/chrome-preamble-search.html'
+
+    def get_context_data(self, **kwargs):
+        label_parts = kwargs.get('label_id', '').split('/')
```

Comment:
This can never contain slashes, right? Also, using named parameters might make this a bit easier to follow.